catalog/core/catalog-core-standardframework/src/main/resources/OSGI-INF/blueprint/delegate-operators.xml

```diff
@@ -105,6 +105,7 @@
         <argument ref="cfOpsMetacard"/>
         <argument ref="cfOpsCatStore"/>
```

Comment:
As mentioned on the class, the `catStore` should be removed from the ctor at this time and set by property instead.